static/img/block1.svg

```diff
@@ -0,0 +1,3 @@
ture 2</div></div></div></foreignObject><text x="360" y="384" fill="#000000" font-family="Helvetica" font-size="12px" text-anchor="middle">Signature 2</text></switch></g><rect x="510" y="120" width="220" height="300" fill="#fff2cc" stroke="#d6b656" pointer-events="all"/><rect x="580" y="130" width="80" height="20" fill="none" stroke="none" pointer-events="all"/><g transform="translate(-0.5 -0.5)"><switch><foreignObject style="overflow: visible; text-align: left;" pointer-events="none" width="100%" height="100%" requiredFeatures="http://www.w3.org/TR/SVG11/feature#Extensibility"><div xmlns="http://www.w3.org/1999/xhtml" style="display: flex; align-items: unsafe center; justify-content: unsafe center; width: 78px; height: 1px; padding-top: 140px; margin-left: 581px;"><div style="box-sizing: border-box; font-size: 0; text-align: center; "><div style="display: inline-block; font-size: 12px; font-family: Helvetica; color: #000000; line-height: 1.2; pointer-events: all; white-space: normal; word-wrap: normal; ">Proof</div></div></div></foreignObject><text x="620" y="144" fill="#000000" font-family="Helvetica" font-size="12px" text-anchor="middle">Proof</text></switch></g><rect x="540" y="170" width="160" height="60" rx="9" ry="9" fill="#ffe6cc" stroke="#d79b00" pointer-events="all"/><g transform="translate(-0.5 -0.5)"><switch><foreignObject style="overflow: visible; text-align: left;" pointer-events="none" width="100%" height="100%" requiredFeatures="http://www.w3.org/TR/SVG11/feature#Extensibility"><div xmlns="http://www.w3.org/1999/xhtml" style="display: flex; align-items: unsafe center; justify-content: unsafe center; width: 158px; height: 1px; padding-top: 200px; margin-left: 541px;"><div style="box-sizing: border-box; font-size: 0; text-align: center; "><div style="display: inline-block; font-size: 12px; font-family: Helvetica; color: #000000; line-height: 1.2; pointer-events: all; white-space: normal; word-wrap: normal; ">private key 2</div></div></div></foreignObject><text x="620" y="204" fill="#000000" font-family="Helvetica" font-size="12px" text-anchor="middle">private key 2</text></switch></g><path d="M 210 310 L 235 310 L 235 380 L 253.63 380" fill="none" stroke="#000000" stroke-miterlimit="10" pointer-events="stroke"/><path d="M 258.88 380 L 251.88 383.5 L 253.63 380 L 251.88 376.5 Z" fill="#000000" stroke="#000000" stroke-miterlimit="10" pointer-events="all"/><path d="M 460 310 L 620 310 L 620 236.37" fill="none" stroke="#000000" stroke-miterlimit="10" pointer-events="stroke"/><path d="M 620 231.12 L 623.5 238.12 L 620 236.37 L 616.5 238.12 Z" fill="#000000" stroke="#000000" stroke-miterlimit="10" pointer-events="all"/></g><switch><g requiredFeatures="http://www.w3.org/TR/SVG11/feature#Extensibility"/><a transform="translate(0,-5)" xlink:href="https://www.diagrams.net/doc/faq/svg-export-text-problems" target="_blank"><text text-anchor="middle" font-size="10px" x="50%" y="100%">Viewer does not support full SVG 1.1</text></a></switch></svg>
```

Comment:
For consistency, the arrow from "root pk" should point to "signature 0", not to the whole block (same in the previous diagram)